.travis.yml

```diff
@@ -7,7 +7,8 @@
         "rvm install 2.2",
         "rvm use 2.2 --fuzzy",
         "export GEMDIR=$(rvm gemdir)",
-        "echo \"ruby=$(basename $GEMDIR) jekyll=$JEKYLL_VERSION\" > pseudo_Gemfile.lock"
+        "echo \"ruby=$(basename $GEMDIR) jekyll=$JEKYLL_VERSION\" > pseudo_Gemfile.lock",
+        "npm install -g npm@3.10.7"
```

Comment:
I made it as separate PR #393 so after accepting that, I will remove these lines